client/connector_list.go

```diff
@@ -68,3 +68,24 @@ func (cli *VanClient) ConnectorList(ctx context.Context) ([]types.LinkStatus, er
 	}
 	return links, nil
 }
+
+func (cli *VanClient) getLinkStatusByNamespace(namespace string) (map[string]*types.LinkStatus, error) {
+	mapLinks := make(map[string]*types.LinkStatus)
+	secrets, err := cli.KubeClient.CoreV1().Secrets(namespace).List(metav1.ListOptions{LabelSelector: "skupper.io/type in (connection-token, token-claim)"})
+	if err != nil {
+		return nil, err
+	}
+	current, err := cli.getRouterConfig()
+	if err != nil {
+		return nil, err
+	}
+	edge := current.IsEdge()
+	connections, _ := qdr.GetConnections(namespace, cli.KubeClient, cli.RestConfig)
```

Comment:
Suggest checking error returned on GetConnections (e.g. could be router exec or unmarshal issue)

Reply:
Good point, I will check it